src/ps1/ps1_cardman.c

```diff
@@ -73,14 +80,17 @@ void ps1_cardman_open(void) {
     char path[64];
 
     ensuredirs();
-    snprintf(path, sizeof(path), "MemoryCards/PS1/Card%d/Card%d-%d.mcd", card_idx, card_idx, card_chan);
+    if ((card_game_id[0] != 0x00) && (card_idx < IDX_MIN)) {
```

Comment:
the condition is different from the one in `ensuredirs` where it's just checking `card_game_id[0] != 0x00` could that be a problem if there's a mismatch?

Reply:
It's better to have the same check in both positions for consistency.